tests/ulyanov_et_al_2016/test_modules.py

```diff
@@ -76,6 +109,20 @@ def test_ulyanov_et_al_2016_upsample(subtests):
         assert module.mode == "nearest"
 
 
+def test_UlyanovEtAl2016HourGlassBlock(subtests):
+    intermediate = nn.Conv2d(3, 3, 1)
+    sequential = modules.UlyanovEtAl2016HourGlassBlock(intermediate)
+
+    assert isinstance(sequential, modules.SequentialWithOutChannels)
+
+    with subtests.test("modules"):
+        assert len(sequential) == 3
+        sequential_modules = tuple(module for module in sequential.children())
+        assert isinstance(sequential_modules[0], nn.Module)
```

Comment:
```suggestion
        assert isinstance(sequential.down, modules.UlyanovEtAl2016StylizationDownsample)
```

Same for the other two. You could wrap them into individual subtests.